app/index.js

```diff
@@ -50,30 +50,70 @@ var BreiAppGenerator = yeoman.generators.Base.extend({
 	},
 
 	writing: {
-		app: function () {
+		folders: function () {
 			this.dest.mkdir('app');
+
+			// Assembled HTML
 			this.dest.mkdir('app/modules');
+			// Compiled CSS
 			this.dest.mkdir('app/css');
+			// Your scripts
 			this.dest.mkdir('app/js');
-			this.dest.mkdir('app/img');
+			this.dest.mkdir('app/js/plugins');
+			this.dest.mkdir('app/js/modules');
+			this.dest.mkdir('app/js/lib');
```

Comment:
I guess this is an interesting idea. Will this be where bower installs stuff? The split between plugins and bower components was to have bower installed dependancies under bower_components, while plugins contained manually managed scripts.

Will app/js/lib and app/js/plugins reflect this mentality? Did you make and test those changes elsewhere in the project?